toolkit/scripts/pkggen.mk

```diff
@@ -110,7 +120,19 @@ $(graph_file): $(specs_file) $(go-grapher) $(toolchain_rpms)
 		$(if $(filter y,$(ENABLE_MEM_PROFILE)),--enable-mem-prof) \
 		$(if $(filter y,$(ENABLE_TRACE)),--enable-trace) \
 		--timestamp-file=$(TIMESTAMP_DIR)/grapher.jsonl \
-		--output $@
+		--output $@ \
+		$(if $(filter y,$(RESOLVE_CYCLES_FROM_UPSTREAM)), --resolve-cycles-from-upstream) \
```

Comment:
I'm wondering if this would make more sense if the extra parameters are moved into the `ifeq ($(RESOLVE_CYCLES_FROM_UPSTREAM),y)` block above? (We do similar stuff with the `graphpkgfetcher_extra_flags`)

Reply:
Just a word of explanation on my end: I believe that was the initial design in this PR and I asked to refactor it this way - I wasn't aware of `graphpkgfetcher_extra_flags` at that time. To be honest, I still think this way is easier to see how the arguments are used, since they are listed along with other ones. Having said that, I don't have strong opinions here, since we already have the precedent elsewhere.

Reply:
I have no strong feelings as well, we can leave as is in that case.